homework-submission/todoapp/src/Components/todos.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import TodoElement from './TodoElement';
+import { observer, inject } from 'mobx-react';
+
+@inject('todos')
+@observer
+class TodoList extends React.Component {
+  render() {
+    const { todos } = this.props.todos;
+    const list = todos.map((item, i) => <TodoElement key={i} todo={item} />);
```

Comment:
While we have the `id` property, and you are setting it very well, why are you using the `index` as a key ?